MicroProfile-JWT-Auth/tck-runner/pom.xml

```diff
@@ -196,7 +201,25 @@
                             </systemPropertyVariables>
                         </configuration>
                     </plugin>
+                    <plugin>
```

Comment:
Isn't specifying this plugin redundant? What is it doing?

Reply:
It ensures that thearquillian.xml file is picked up. I would have thought that it would run be default but by debugging maven it seems not to be.

Reply:
In that case, would it be helpful to specify it in the root parent pom?

Reply:
No, as it already runs otherwise. This only occurs for this specific submodule.

Reply:
Why does it only not run for this specific module?

Reply:
I don't know. Something weird happens with Maven.

Reply:
Does it break if the resources plugin is specified in the parent pom? I think it would be cleaner if it's possible

Reply:
I don't think it would be cleaner, as this is an issue that only occurs in this module, so my view is that the solution should only be present in this module.